src/redux/actions/composeAction.tsx

```diff
@@ -196,4 +210,69 @@ export const searchSchemaByTagsInfoFailureAction = (
 ): SearchSchemaByTagInfoFailureAction => ({
   type: SEARCH_SCHEMA_BY_TAG_INFO_ACTION_FAILURE,
   payload,
-});
\ No newline at end of file
+});
+
+  // POST COMPOSE NAME AND DESC
+  console.log("getting into action page");
+
+  export const postComposeNameDescRequest = (
+    name:any,
+    description:any
+  ): PostComposeNameDescRequest => ({
+      type: POST_COMPOSE_NAME_DESC,
+      name,
+      description,
+  });
+  
+  export const postComposeNameDescRequestSuccess = (
+    postData: any
+  ): PostComposeNameDescSuccess => ({
+      type: POST_COMPOSE_NAME_DESC_SUCCESS,
+      payload: { postData },
+  
+  });
+  
+  export const postComposeNameDescRequestFailure = (
+    error: any
+  ): PostComposeNameDescFailure => ({
+      type: POST_COMPOSE_NAME_DESC_FAILURE,
+      payload: { error },
+  });
+  
+  
+  // GET COMPOSE NAME AND DESC
+  
+  export const getComposeNameDescRequest = (
+    // uid: any,
+    // name: any,
```

Comment:
Any purpose in keeping it? If not remove commented code

Reply:
Done